src/dune_rules/dune_file.ml

```diff
@@ -2378,6 +2379,10 @@ module Stanzas = struct
       , let+ () = Dune_lang.Syntax.since Section.dune_site_syntax (0, 1)
         and+ t = Plugin.decode in
         [ Plugin t ] )
+    ; ( "melange.emit"
+      , let+ () = Dune_lang.Syntax.since Stanza.syntax (3, 6)
```

Comment:
This should check against the melange syntax

Reply:
https://github.com/ocaml/dune/pull/6349/commits/61cbf2805be9458dc87be664cc6054b36087a669. I understand there's no need to bump the syntax, as we are merely adding functionality here.